WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml

```diff
@@ -0,0 +1,44 @@
+# TestGrid job configuration
+
+version: '0.9'
+emailToList: "integration-builder@wso2.com,rakhitha@wso2.com,prasanna@wso2.com,
+              niron@wso2.com,mushthaq@wso2.com,prasanna@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+    - CentOS-7.5
+    - MySQL-5.7
+    - Oracle-SE2-12.1
+    - SQLServer-SE-14.00
+    - Postgres-10.5
+    - Postgres-12.3
+    - Postgres-13.2
```

Comment:
I am not sure whether this version is also supported by AWS RDS anymore. Please check.